src/js/main.js

```diff
@@ -17,7 +17,14 @@
       templateUrl: 'partials/user-wishes.html'
     })//END OF USER-WISHES
     .when ('/results', {
-      templateUrl: 'partials/results.html'
+      templateUrl: 'partials/results.html',
+      controller: function($http, $scope){
+        $http.get('http://wishcastr-staging.herokuapp.com/products/search.json?query='+$scope.query)
```

Comment:
Also, this would be a good place to use [that `BASE_URL` constant I mentioned in another PR](https://github.com/wishcastr/wishcastr/pull/21#discussion-diff-43895217)